MakieCore/src/basic_plots.jl

```diff
@@ -484,6 +484,8 @@ Plots a marker for each element in `(x, y, z)`, `(x, y)`, or `positions`.
     marker_offset = Vec3f(0)
     "Controls whether the model matrix (without translation) applies to the marker itself, rather than just the positions. (If this is true, `scale!` and `rotate!` will affect the marker."
     transform_marker = false
+    "Sets the font used for character markers."
+    font = automatic
```

Comment:
I only added `font` here with a `to_font(::Automatic) = defaultfont()` method in conversions.jl. So I didn't mirror the `fonts` interface from text. Not sure whether that's the right way to go about it.